lib/gen/antlrv4/ts-antlrv4/package.json

```
@@ -0,0 +1,20 @@
+{
+  "name": "@webstatus.dev/ts-antlrv4",
+  "version": "1.0.0",
+  "description": "TypeScript ANTLR4 generated code for webstatus.dev",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "license": "Apache-2.0",
+  "scripts": {
+    "build": "tsc",
+    "clean": "rimraf dist"
```

```suggestion
    "build": "tsc",
    "clean": "rm -rf dist"
```

We should rely on the system rm command.